src/test/ui/async-await/suggest-missing-await.rs

```diff
@@ -0,0 +1,38 @@
+// edition:2018
```

Comment:
nit: you could add `// run-rustfix` to this and the test will have the suggestions applied and checked to be sure that the code with suggestions will now compile. You'd need to move `foo2` to another file so that with the suggestions applied, the code would actually compile.